packages/core/src/hosted-form/iframe-content/hosted-input.ts

```diff
@@ -52,6 +54,10 @@ export default class HostedInput {
             HostedFieldEventType.SubmitRequested,
             this._paymentHandler.handle,
         );
+        this._eventListener.addListener(
+            HostedFieldEventType.VaultingRequested,
```

Comment:
Where does this event get emitted?

Reply:
I’m not sure that I understood correctly about this one.
I understand that you recommended to move client logic to the SDK as we have for payment hosted fields. So in this case I need to create a vaulting/payment strategy(some separate package) to expose interface to working with hosted fields with client. Am I correct?
In this case I'll add SDK to microapp and would import only vaulting package to the microapp.

Reply:
> In this case I'll add SDK to microapp and would import only vaulting package to the microapp.

Yes, that'd be my recommendation. Export an interface from the SDK. When it's called, it'll emit the `HostedFieldEventType.VaultingRequested` event to submit the form to the vaulting endpoint.